cores/esp8266/core_esp8266_postmortem.cpp

```diff
@@ -82,10 +82,15 @@ static void ets_printf_P(const char *str, ...) {
     }
 }
 
+#define FAKE_REASON_NONE 255
+#define FAKE_REASON_USER 254
+static int fake_rst_reason = FAKE_REASON_NONE;
+
 void __wrap_system_restart_local() {
     register uint32_t sp asm("a1");
     uint32_t sp_dump = sp;
 
+#if 0
```

Comment:
We should drop these lines completely.  Git will preserve the old info if needed.